example/road-traffic/road-traffic-generators/src/main/java/uk/gov/gchq/gaffer/traffic/generator/RoadTrafficDataLoader.java

```diff
@@ -121,7 +121,7 @@ public static void main(final String[] args) {
             dataLoader.load(new File(dataFile));
             LOGGER.info("Data has been loaded");
         } catch (final Exception e) {
-            LOGGER.info("Unable to load data: " + e.getMessage());
```

Comment:
This file shouldn't have been changed. I think this will have been caused by adding something and not reverting correctly.

Reply:
Correct, this was incorrectly reverted, have removed change and fixed copyright